example/main.go

```diff
@@ -88,14 +90,15 @@ func Main() int {
 		// error already logged
 		return 1
 	}
-	fmt.Fprintf(t.Out, "Terminal is open\nis valid %t\nuse exit or ^D or ^C to exit\n", t.IsTerminal())
+	fmt.Fprintf(t.Out, "Terminal is open\nis valid %t\nuse exit or ^D or 3 ^C to exit\n", t.IsTerminal())
```

Comment:
Here it's ^C but https://github.com/fortio/terminal/pull/20#discussion_r1739901570